nft_ingester/src/api/dapi/asset.rs

```diff
@@ -135,17 +136,18 @@ fn asset_selected_maps_into_full_asset(
     options: &Options,
 ) -> Option<FullAsset> {
     if !options.show_unverified_collections {
-        if let Some(collection_data) = asset_selected_maps
-            .asset_complete_details
-            .get(id)
-            .and_then(|a| a.collection.clone())
-        {
-            if !collection_data.is_collection_verified.value {
-                return None;
+        if let Some(asset_complete_details) = asset_selected_maps.asset_complete_details.get(id) {
+            if let Some(asset_static_details) = &asset_complete_details.static_details {
+                // collection itself cannot have a collection
+                // TODO!: should we also include in this check FungibleToken?
+                if &asset_static_details.specification_asset_class != &SpecificationAssetClass::MplCoreCollection {
```

Comment:
Can other protocols have collections as separate assets/accounts? If so, maybe we can check it somehow (i mean check if the account/asset represents a collection). Maybe it sounds like a separate ticket, so there are no actions required now